maintainers.txt

```diff
@@ -0,0 +1 @@
+nlisgo
```

Comment:
I think you should add yourself to this also Dave, as there is a fair amount of HTML in there.